config/initializers/lookbook.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# lookbook workaround for phlex components
+if defined?(Lookbook)
```

Comment:
Lookbook has some compatibility issues with Phlex that will be fixed in Phlex 2.0.